lms/djangoapps/grades/models.py

```diff
@@ -257,8 +257,14 @@ def save_grade(cls, **kwargs):
                     usage_key=usage_key,
                     defaults=kwargs,
                 )
+            log.info("persistent subsection grade successfully saved: {0}".format(unicode(grade)))
         except IntegrityError:
             cls.update_grade(user_id=user_id, usage_key=usage_key, **kwargs)
+            log.info("integrity error trying to save grade for user {0}, usage key {1}, defaults {2}".format(
```

Comment:
I'd lean towards `warning` for this log statement.  Integrity errors shouldn't be too common, and we want them to stand out, so we know how often it happens.